modules/cluster-logging-collector-legacy-syslog.adoc

```diff
@@ -81,27 +81,26 @@ To configure {product-title} to forward logs using the legacy configuration meth
 @type <type> <1>
 remote_syslog <syslog-server> <2>
 port 514 <3>
-hostname <host> <4>
-remove_tag_prefix <prefix> <5>
+hostname ${hostname}
```

Comment:
Do you define this value somewhere else? If the user needs to manually pick it here, the previous format was correct. If it's defined elsewhere, it would be useful to know where.

Reply:
> Do you define this value somewhere else? If the user needs to manually pick it here, the previous format was correct. If it's defined elsewhere, it would be useful to know where.

@jcantrill @r2d2rnd Please confirm this should appear exactly as shown: That this is an environment variable, not a user-replaceable value.

Reply:
Good question, Kathryn. Although `${hostname}` looks like a user-replaceable value, it should appear as shown on lines 29 and 84 as asserted by @r2d2rnd in https://issues.redhat.com/browse/RHDEVDOCS-2604.